spritesh.gemspec

```diff
@@ -0,0 +1,12 @@
+Gem::Specification.new do |s|
+  s.name        = "spritesh"
+  s.version     = "1.0.7"
+  s.summary     = "Build an SVG sprite from a folder of SVG files."
+  s.description = "A shell script to build an SVG sprite from a folder of SVG files."
```

Comment:
Any chance this file could read from package.json? Would avoid having all this information at 2 different places.